include/meta/meta.hpp

```diff
@@ -1990,8 +1990,9 @@ namespace meta
             template <typename... T, typename V>
             struct find_index_<list<T...>, V>
             {
-#if defined(__clang__) && __clang_major__ < 6
+#if (defined(__clang__) && __clang_major__ < 6) || defined(__apple_build_version__)
                 // Explicitly specify extent to avoid https://llvm.org/bugs/show_bug.cgi?id=28385
+                // Track Apple's version here: https://en.wikipedia.org/wiki/Xcode#Toolchain_Versions
```

Comment:
Having now actually looked at this page, there's no useful information there: I don't think we need these Wikipedia links. We like having links to trackable bugs for workarounds like this so the workarounds can be scoped to the buggy compiler versions and eventually dropped along with support for said buggy compiler versions. I vaguely recall hearing someone mention that Apple doesn't have a publicly accessible bug tracking database so there may be no way for us to get such a link in this case. If you happen to know that I'm mistaken, then please speak up, but otherwise we should simply remove the wikipedia links and hope that someone some day will tell us this bug has been fixed in some Apple clang version.

Reply:
Alright, I somehow assumed the link provided a mapping of Apple's LLVM version to the official one. But I was apparently mistaken as it doesn't include newer versions.